.github/workflows/pr-build.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        java: [ '11' , '12' ]
+        java: [ '8' , '12' ]
```

Comment:
Don't we want to abandon Java 8 ASAP?

Reply:
Yeah, I've just kept it to be sure it can still work thill we decide to abandon Java 11